README.md

```diff
@@ -77,6 +80,21 @@ data:
   concurrency: "10"
 ```
 
+### Enable Persistent Volume extension in Knative
+
+In order to use `volumeMounts` in `Capp`, it is needed to configure `Knative Serving` to support volumes. This is done by adding the following lines to the `ConfigMap` of name `config-features` in the `Knative Serving` namespace:
```

Comment:
instead of `it is needed to configure ....` change it knative serving needs to be configured ...

instead of name change it to `named`